test/image/mocks/gl2d_parcoords_out-of-range_selected-none.json

```diff
@@ -0,0 +1,63 @@
+{
```

Comment:
This one renders currently as:

![image](https://user-images.githubusercontent.com/6675409/59610319-225ee400-90e7-11e9-8289-48969d9fee94.png)

Reply:
@etpinard Thanks a million for documenting this one two!